pkgs/development/python-modules/airlift/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, poetry-core, argparse, halo, pyyaml, hiyapyco, jinja2, dotmap, requests
+, kubernetes-helm, kind, docker, pythonOlder, buildPythonPackage
+, fetchFromGitHub }:
+
+buildPythonPackage rec {
+
+  disabled = pythonOlder "3.9";
+  pname = "airlift";
+  format = "pyproject";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "jl178";
+    repo = "airlift";
+    rev = "v${version}";
+    sha256 = "ZQ+YCMXQBM5Gb5O1jDQfKBeS1WxFKrtSRelgj65+f9U=";
+  };
+  nativeBuildInputs = [ poetry-core ];
+  propagatedBuildInputs = [
+    argparse
+    halo
+    pyyaml
+    hiyapyco
+    jinja2
+    dotmap
+    requests
+    kubernetes-helm
+    kind
+    docker
```

Comment:
I don't think we need these packages.
https://github.com/jl178/airlift/blob/d0cd62f85b5694f81f11298d5d34bd94001210bd/pyproject.toml#L9-L17

Reply:
If those packages are needed for the CLI to function properly, where should I put them? They are not python specific packages, but the CLI calls on those binaries (ex, it will run `helm install` or `docker ps`)

Reply:
Could you please add them to `buildInputs`?